src/app/functions/server/generateJWT.ts

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import jwt from "jsonwebtoken";
+import { redisClient, REDIS_JWT_KEY_PREFIX } from "../../../db/redis/client";
+import { isAdmin } from "../../api/utils/auth";
+import { logger } from "./logging";
+const rClient = redisClient();
+
+// We'll need a key ID to identify the key used for signing
+const KEY_ID = process.env.MOSCARY_JWT_KEY_ID || "moscary-jwt-key";
+
+export async function getMoscaryJWT(fxaUid: string, email: string) {
+  const REDIS_KEY = `${REDIS_JWT_KEY_PREFIX}${fxaUid}`;
```

Comment:
Recommend using session identifier here since otherwise there's no way to invalidate the cache before TTL ends. For example if we make someone an admin, they can log out and log in to get access to their new role immediately versus having to wait for the timeout period (or otherwise implementing cache invalidation)

Reply:
Good catch! Fixed, thanks